site/content/blog/2025-02-17-portswigger-lab-walkthrough-broken-brute-force-protection-ip-block/index.md

```diff
@@ -0,0 +1,793 @@
+---
+title: "PortSwigger Lab Walkthrough: Broken Brute-Force Protection, IP Block"
+summary: >
+  Walkthrough for the PortSwigger lab, "Broken brute-force protection, IP block".
+# images:
+# - 
```

Comment:
It would be great if you could some up with at least one so that it can be used for the social preview